src/kubelet/kubelet_test.go

```diff
@@ -28,10 +26,11 @@ func TestScraper(t *testing.T) {
 	asserter := testutil.NewAsserter().
 		Using(metric.KubeletSpecs).
 		Silently().
```

Comment:
I'd get rid of `Silently()` (in fact I had serious doubts about implementing it in the first place) as it will hide what we are not testing from the test output.